File: classes/en/Consulta.php
<?php
    class ENConsulta{  
        var $idConsulta;
        var $tiConsulta;
        var $teConsulta;
        var $feConsulta;
        var $respondeA;
        var $suscripcion;

        function setIdConsulta($idConsulta) { $this->idConsulta = $idConsulta; }
        function getIdConsulta() { return $this->idConsulta; }
        function setTiConsulta($tiConsulta) { $this->tiConsulta = $tiConsulta; }
        function getTiConsulta() { return $this->tiConsulta; }
        function setTeConsulta($teConsulta) { $this->teConsulta = $teConsulta; }
        function getTeConsulta() { return $this->teConsulta; }
        function setFeConsulta($feConsulta) { $this->feConsulta = $feConsulta; }
        function getFeConsulta() { return $this->feConsulta; }
        function setRespondeA($respondeA) { $this->respondeA = $respondeA; }
        function getRespondeA() { return $this->respondeA; }
        function setSuscripcion($suscripcion) { $this->suscripcion = $suscripcion; }
        function getSuscripcion() { return $this->suscripcion; }
   
    }
?>

File: Android/RapiMoncha/src/una/android/rapimoncha/entidades/Consulta.java
package una.android.rapimoncha.entidades;

import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Calendar;

/**
 * Created by Lester on 31/03/2015.
 */
public class Consulta {
    private int idConsulta;
    private String tiConsulta;
    private String teConsulta;
    private Calendar feConsulta;
    private Consulta respondeA;
    private Subcripcion subcripcion;


    public Consulta() {
    }

    public int getIdConsulta() {
        return idConsulta;
    }

    public void setIdConsulta(int idConsulta) {
        this.idConsulta = idConsulta;
    }

    public String getTiConsulta() {
        return tiConsulta;
    }

    public void setTiConsulta(String tiConsulta) {
        this.tiConsulta = tiConsulta;
    }

    public String getTeConsulta() {
        return teConsulta;
    }

    public void setTeConsulta(String teConsulta) {
        this.teConsulta = teConsulta;
    }

    public Calendar getFeConsulta() {
        return feConsulta;
    }

    public void setFeConsulta(Calendar feConsulta) {
        this.feConsulta = feConsulta;
    }

    public Consulta getRespondeA() {
        return respondeA;
    }

    public void setRespondeA(Consulta respondeA) {
        this.respondeA = respondeA;
    }

    public Subcripcion getSubcripcion() {
        return subcripcion;
    }

    public void setSubcripcion(Subcripcion subcripcion) {
        this.subcripcion = subcripcion;
    }
    public String getDateTimeConsulta(){
         DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
         return dateFormat.format(feConsulta.getTime());
    }
}
